roles/upgrade/defaults/main.yml

```diff
@@ -165,7 +165,7 @@ check_leapp_analysis_results: true
 
 kernel_modules_to_unload_before_upgrade: []
 
-post_7_to_8_python_interpreter: /usr/libexec/platform-python
+post_upgrade_python_interpreter: "{{ (ansible_distribution_major_version == '7' and post_7_to_8_python_interpreter is defined) | ternary(post_7_to_8_python_interpreter, '/usr/libexec/platform-python') }}"
```

Comment:
Are you sure this is true for both 8 to 9 and 9 to 10?  The leapp-upgrade task below will now set this `when: ansible_distribution_major_version | int >= 7`

Reply:
I've checked and the `/usr/libexec/platform-python` is present on RHEL 7+